packages/shopify-codemod/transforms/catch-undefined-to-catch-error.js

```diff
@@ -0,0 +1,17 @@
+export default function catchUndefinedToCatchError({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+  function hasNoErrorReferences(path) {
+    const usage = j(path).find(j.Identifier, {name: 'error'});
```

Comment:
I could have done something more involved with scope/binding info here, but ultimately `error` shadowing isn't an issue in `shopify/shopify`, and the linter will flag it if it becomes an issue.